### ISSUE TITLE
Issue in enum documentation

### ISSUE THREAD

**Issue by vstinner** [python/cpython#73852]
BPO | [29666](https://bugs.python.org/issue29666)
--- | :---
Nosy | @vstinner, @ethanfurman, @DimitrisJim, @kartikanand
PRs | <li>python/cpython#358</li><li>python/cpython#367</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = <Date 2017-03-02.11:48:04.406>
created_at = <Date 2017-02-27.13:48:31.073>
labels = ['easy', '3.7', 'docs']
title = 'Issue in enum documentation'
updated_at = <Date 2017-03-02.11:48:04.405>
user = 'https://github.com/vstinner'
```

bugs.python.org fields:
```python
activity = <Date 2017-03-02.11:48:04.405>
actor = 'vstinner'
assignee = 'docs@python'
closed = True
closed_date = <Date 2017-03-02.11:48:04.406>
closer = 'vstinner'
components = ['Documentation']
creation = <Date 2017-02-27.13:48:31.073>
creator = 'vstinner'
dependencies = []
files = []
hgrepos = []
issue_num = 29666
keywords = ['easy']
message_count = 5.0
messages = ['288642', '288643', '288647', '288721', '288797']
nosy_count = 5.0
nosy_names = ['vstinner', 'docs@python', 'ethan.furman', 'Jim Fasarakis-Hilliard', 'exqu17']
pr_nums = ['358', '367']
priority = 'normal'
resolution = 'fixed'
stage = 'resolved'
status = 'closed'
superseder = None
type = None
url = 'https://bugs.python.org/issue29666'
versions = ['Python 3.6', 'Python 3.7']
```

</p></details>

**Comment by vstinner** [comment]
https://docs.python.org/dev/library/enum.html

"This module defines four enumeration classes that can be used to define unique sets of names and values: Enum, IntEnum, and IntFlags"

Enum, IntEnum, IntFlag: I count 3 classes, no 4?

Moreover, the correct class name is IntFlag, not IntFlags. Or did I miss something?

```pycon
haypo@selma$ ./python
Python 3.7.0a0 (default, Feb 25 2017, 04:30:32) 
>>> import enum
>>> enum.IntFlags
...
AttributeError: module 'enum' has no attribute 'IntFlags'
```

**Comment by b78688f9-6435-4192-9fe9-2f86f7e61fcf** [comment]
Yup, `IntFlags` is a typo, that's why the reference to it doesn't show too. 

Also, the fourth enum must be `Flag` which must of been omitted when it was added in `3.6`.

**Comment by ethanfurman** [comment]
`Flag` and `IntFlag` are indeed the correct (and missing) names.

**Comment by 1b923c99-7ee8-4252-896b-4877d8e10a5a** [comment]
Hi I've edited the documentation to correct this and submitted a PR.

**Comment by vstinner** [comment]
Doc has been fixed in Python 3.6 and 3.7. Thanks!
